sdk/eventhub/Azure.Messaging.EventHubs/api/Azure.Messaging.EventHubs.netstandard2.0.cs

```diff
@@ -415,6 +415,7 @@ public abstract partial class EventProcessor<TPartition> where TPartition : Azur
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string ToString() { throw null; }
         protected virtual System.Threading.Tasks.Task UpdateCheckpointAsync(string partitionId, long offset, long? sequenceNumber, System.Threading.CancellationToken cancellationToken) { throw null; }
+        protected internal virtual System.Threading.Tasks.Task ValidateStartupAsync(bool async, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
This had previously been `private` and the processor client had to do some odd gymnastics to extend the validation behavior.   This allows derived processors to more easily participate in validation at startup and mock for testing.